src/metabase/troubleshooting.clj

```diff
@@ -40,6 +40,7 @@
                                        :jdbc-driver {:name    (.getDriverName metadata)
                                                      :version (.getDriverVersion metadata)}}))
     :run-mode                     (config/config-kw :mb-run-mode)
+    :plan-alias (or (some-> (premium-features/premium-embedding-token) premium-features/fetch-token-status :plan-alias) "")
```

Comment:
can you align this with the rest of the map?

Reply:
also do we need the empty string fallback? I don't know exactly how this map is used but I assume it should be able to handle a `nil` value